docs/how-to-guides/themes/create-block-theme.md

```diff
@@ -27,6 +28,7 @@ inside a folder called `templates`.
 
 The theme may optionally include a `functions.php` file and a [theme.json file](/docs/how-to-guides/themes/theme-json.md) to manage global styles.
 Template parts are optional. If they are included they must be placed inside a `parts` folder.
+The theme may optionally include a `styles` folder to provide global styles presets.
```

Comment:
It would be a good idea maybe to make `global styles presets` a link? 🤔

Reply:
+1 I think it would be smart to add more context here as otherwise it's not clear why one might do this (and it's such a neat thing to be able to do!).